worker/orca-agent/src/server/services/task_service.py

```
@@ -841,6 +863,60 @@ def create_aggregator_repo(task_id):
+        max_retries = 10
+        retry_delay = 2  # seconds
+        for attempt in range(max_retries):
+            try:
+                fork.get_commits().get_page(0)
+                logger.info(f"Fork is ready after {attempt + 1} attempts")
+                break
+            except Exception as e:
+                if attempt == max_retries - 1:
+                    raise Exception(
+                        f"Fork not ready after {max_retries} attempts: {str(e)}"
+                    )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)